libstemmer/modules.txt

```diff
@@ -28,6 +28,8 @@ portuguese      UTF_8,ISO_8859_1        portuguese,pt,por
 romanian        UTF_8,ISO_8859_2        romanian,ro,rum,ron
 russian         UTF_8,KOI8_R            russian,ru,rus
 spanish         UTF_8,ISO_8859_1        spanish,es,esl,spa
+catalan         UTF_8,ISO_8859_1        catalan,ca,cat
+basque          UTF_8,ISO_8859_1        basque,eu,eus,baq
```

Comment:
Did you intentionally list these languages after Spanish?

This makes some sense from geographical point of view, but in my opinion it’s better to have this list sorted alphabetically.

Reply:
Indeed, thanks for spotting - I've pushed 2aa2809f67332b7f5c3494a741cd0390a6b40cff to fix that and 0e0d307b082c0c5758ea7f89bb9b3d3dd9b215e1 to note this in the instructions.

Reply:
Thanks for the correction!